source/extensions/common/wasm/wasm.h

```diff
@@ -316,8 +327,9 @@ class Wasm : public Envoy::Server::Wasm,
 
   WasmContextCall7Void onHttpCallResponse_;
 
-  WasmContextCall0Void onLog_;
   WasmContextCall0Void onDone_;
+  WasmContextCall0Void onLog_;
```

Comment:
Does the order of onDone_ and onLog_ matter? Not in syntax,  but I am wondering if there is any implicitly indication.

Reply:
onDone indicates that the stream has closed.  It is an artifact of envoy (AFAICT) that the order is onDone -> onLog.  I had to test it to see which way it would go.